CODE_OF_CONDUCT.md

```diff
@@ -1,53 +1,113 @@
+reporter of any incident.
 
-4. **Personal Attacks:** Personal attacks, insults, or disrespectful comments directed at other participants.
+## Enforcement Guidelines
 
-5. **Invasion of Privacy:** Sharing personal information, such as addresses, phone numbers, or private conversations, without consent.
+Community leaders will follow these Community Impact Guidelines in determining
+the consequences for any action they deem in violation of this Code of Conduct:
 
-6. **Sexual Harassment:** Unwanted sexual advances, comments, or conduct.
+### 1. Correction
 
-## Enforcement
+**Community Impact**: Use of inappropriate language or other behavior deemed
+unprofessional or unwelcome in the community.
+
+**Consequence**: A private, written warning from community leaders, providing
+clarity around the nature of the violation and an explanation of why the
+behavior was inappropriate. A public apology may be requested.
+
+### 2. Warning
+
+**Community Impact**: A violation through a single incident or series of
+actions.
+
+**Consequence**: A warning with consequences for continued behavior. No
+interaction with the people involved, including unsolicited interaction with
+those enforcing the Code of Conduct, for a specified period of time. This
+includes avoiding interactions in community spaces as well as external channels
+like social media. Violating these terms may lead to a temporary or permanent
+ban.
+
+### 3. Temporary Ban
 
-1. **Investigation:** All reports of Code of Conduct violations will be promptly and thoroughly investigated by the event organizers.
+**Community Impact**: A serious violation of community standards, including
+sustained inappropriate behavior.
 
-2. **Consequences:** Participants who engage in unacceptable behavior may face consequences, including warnings, temporary suspension, or permanent exclusion from the event, at the discretion of the event organizers.
+**Consequence**: A temporary ban from any sort of interaction or public
+communication with the community for a specified period of time. No public or
+private interaction with the people involved, including unsolicited interaction
+with those enforcing the Code of Conduct, is allowed during this period.
+Violating these terms may lead to a permanent ban.
 
-3. **Appeals:** Participants have the right to appeal any decision made by the event organizers regarding Code of Conduct violations.
+### 4. Permanent Ban
 
-## Acknowledgment
+**Community Impact**: Demonstrating a pattern of violation of community
+standards, including sustained inappropriate behavior, harassment of an
+individual, or aggression toward or disparagement of classes of individuals.
 
-By participating in Hacktoberfest, you agree to abide by this Code of Conduct. We are committed to providing a positive and inclusive experience for all, and your cooperation helps us achieve that goal.
+**Consequence**: A permanent ban from any sort of public interaction within the
```

Comment:
How can we ban someone for any sort of public interaction that to for a repository in Github?